app/src/main/java/com/eveningoutpost/dexdrip/Services/G5CollectionService.java

```diff
@@ -266,13 +266,11 @@ public void onDestroy() {
 
         Log.d(TAG, "onDestroy");
         SharedPreferences sharedPrefs = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
-        if (sharedPrefs.getBoolean("use_wear_connectG5", false)) {//KS
```

Comment:
Is this a fix to something which previously got broken? Can you explain the significance of this change just so I understand how it relates?